repository/d.json

```diff
@@ -146,9 +146,10 @@
 			]
 		},
 		{
-			"name": "DarkDonut Forest Coders",
-			"details": "https://github.com/DarkDonut-theme/forest_coders_subime",
-			"description": "Forest Color Scheme for Green lovers 🎃 ",
+			"name": "DarkDonut Color Schemes",
+			"previous_names": ["DarkDonut Forest Coders"],
+			"details": "https://github.com/DarkDonut-theme/DarDonut_sublime",
+			"description": "A set of Color Scheme for Sublime Text 3 that fits your flavour! 🎃",
```

Comment:
I would prefer if you updated the description of the GitHub repository, and then remove the "description" key here. It is not required, we will take the GitHub description instead. And that will allow you to change it at will without having to change the package control entry.

Reply:
Done and Thanks 🙂